Userland/Libraries/LibWeb/CMakeLists.txt

```diff
@@ -353,295 +352,11 @@ set(SOURCES
-libweb_js_wrapper(SVG/SVGPathElement)
-libweb_js_wrapper(SVG/SVGPolygonElement)
-libweb_js_wrapper(SVG/SVGPolylineElement)
-libweb_js_wrapper(SVG/SVGRectElement)
-libweb_js_wrapper(SVG/SVGSVGElement)
-libweb_js_wrapper(SVG/SVGTextContentElement)
-libweb_js_wrapper(Selection/Selection)
-libweb_js_wrapper(UIEvents/FocusEvent)
-libweb_js_wrapper(UIEvents/KeyboardEvent)
-libweb_js_wrapper(UIEvents/MouseEvent)
-libweb_js_wrapper(UIEvents/UIEvent)
-libweb_js_wrapper(URL/URL)
-libweb_js_wrapper(URL/URLSearchParams ITERABLE)
-libweb_js_wrapper(WebSockets/WebSocket)
-libweb_js_wrapper(XHR/ProgressEvent)
-libweb_js_wrapper(XHR/XMLHttpRequest)
-libweb_js_wrapper(XHR/XMLHttpRequestEventTarget)
-
-invoke_generator(
-    "MediaFeatureID"
-    Lagom::GenerateCSSMediaFeatureID
-    "${CMAKE_CURRENT_SOURCE_DIR}/CSS/MediaFeatures.json"
-    ""
-    "CSS/MediaFeatureID.h"
-    "CSS/MediaFeatureID.cpp"
-    arguments -j "${CMAKE_CURRENT_SOURCE_DIR}/CSS/MediaFeatures.json"
-)
-
-invoke_generator(
-    "PropertyID"
-    Lagom::GenerateCSSPropertyID
-    "${CMAKE_CURRENT_SOURCE_DIR}/CSS/Properties.json"
-    ""
-    "CSS/PropertyID.h"
-    "CSS/PropertyID.cpp"
-    arguments -j "${CMAKE_CURRENT_SOURCE_DIR}/CSS/Properties.json"
-)
-
-invoke_generator(
-    "ValueID"
-    Lagom::GenerateCSSValueID
-    "${CMAKE_CURRENT_SOURCE_DIR}/CSS/Identifiers.json"
-    ""
-    "CSS/ValueID.h"
-    "CSS/ValueID.cpp"
-    arguments -j "${CMAKE_CURRENT_SOURCE_DIR}/CSS/Identifiers.json"
-)
-
-add_custom_command(
-    OUTPUT CSS/DefaultStyleSheetSource.cpp
-    COMMAND "${CMAKE_COMMAND}" -E make_directory CSS
-    COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/Scripts/GenerateStyleSheetSource.sh" default_stylesheet_source "${CMAKE_CURRENT_SOURCE_DIR}/CSS/Default.css" > CSS/DefaultStyleSheetSource.cpp.tmp
-    COMMAND "${CMAKE_COMMAND}" -E copy_if_different CSS/DefaultStyleSheetSource.cpp.tmp CSS/DefaultStyleSheetSource.cpp
-    COMMAND "${CMAKE_COMMAND}" -E remove CSS/DefaultStyleSheetSource.cpp.tmp
-    VERBATIM
-    DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/Scripts/GenerateStyleSheetSource.sh"
-    MAIN_DEPENDENCY CSS/Default.css
-)
-add_custom_target(generate_DefaultStyleSheetSource.cpp DEPENDS CSS/DefaultStyleSheetSource.cpp)
-add_dependencies(all_generated generate_DefaultStyleSheetSource.cpp)
-
-add_custom_command(
-    OUTPUT CSS/QuirksModeStyleSheetSource.cpp
-    COMMAND "${CMAKE_COMMAND}" -E make_directory CSS
-    COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/Scripts/GenerateStyleSheetSource.sh" quirks_mode_stylesheet_source "${CMAKE_CURRENT_SOURCE_DIR}/CSS/QuirksMode.css" > CSS/QuirksModeStyleSheetSource.cpp.tmp
-    COMMAND "${CMAKE_COMMAND}" -E copy_if_different CSS/QuirksModeStyleSheetSource.cpp.tmp CSS/QuirksModeStyleSheetSource.cpp
-    COMMAND "${CMAKE_COMMAND}" -E remove CSS/QuirksModeStyleSheetSource.cpp.tmp
-    VERBATIM
-    DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/Scripts/GenerateStyleSheetSource.sh"
-    MAIN_DEPENDENCY CSS/Default.css
+set(GENERATED_SOURCES
```

Comment:
What's this for? Setting GENERATED_SOURCES after we do `add_library` in `serenity_lib` does nothing.

Reply:
This was indeed a mistake (and was the thing causing issues in CI). It worked on my machine because the compiled files were already there, but needed to be moved around properly when rebuilding from scratch. Fixed!